ansible_base/__init__.py

```diff
@@ -0,0 +1,18 @@
+try:
+    from ._version import __version__, __version_tuple__  # noqa: F401
+except Exception:
+    import datetime
+    import subprocess
+    from typing import Tuple, Union
+
+    calver_now = datetime.datetime.now().strftime("%Y.%m.%d")
+    shaw = subprocess.check_output(['git', 'rev-parse', '--short', 'HEAD']).decode('ascii').strip()
```

Comment:
This doesn't feel right. And after reading further I am left wondering why we are doing any of the stuff in this file.